### ISSUE TITLE
Add subprojects/studios feature

### ISSUE THREAD

**Issue by saerdnaer** [crs-tools/tracker#243]
Aktuelle Property-Vererebungshierarchie:

* Event-Projekt für die Konferenz
  * Meta-Ticket (Fahrplan-ID & -Metadaten)
    * Recording-Ticket (Sprache, etc)
       * Master-Encoding-Ticket (MP4)
       * weitere Encoding-Tickets für die Subformate: WebM MP3, Opus, SD-Formate, etc.

Wunsch-Hierarchie für dezentrale Konferenzen wie rC3:

Globale Properties aus Projekt 0
* Event-Projekt für die Konferenz
  * Studio-Projekt z.B. c3s
    * Meta-Ticket (Fahrplan-ID & -Metadaten)
       * Recording-Ticket (Sprache, etc)
         * Master-Encoding-Ticket (MP4)
         * weitere Encoding-Tickets für die Subformate: WebM MP3, Opus, SD-Formate, etc.

**Comment by CodeFreezr** [comment]
Aus Sicht eines Studios, wäre es gummigut ein Projekt zu haben, wo wir keine Angst haben müssen ein Setting für ein anderes Studio zu überschreiben. Was darüber an Abstraktionen Sinnvoll wäre, wage ich nicht zu beurteilen, aber gründsätzlich klingt global -> series -> event -> studio imho vielversprechend.

Und vielleicht könnten man beim erzeugen einfach ein Art "same as" ergänzen. So könnten z.b. ein neues Studio-Projekt für z.b. ccch oder r3s von einem vorherigem angelegt werden. 

Vielleicht ergänzend mit einer kleinen Liste an Projekt-Templates, z.b. "Tracker-Assisted Cut", "Video-Download", ...

**Comment by jjeising** [comment]
Was genau soll das denn abdecken? Nur eine Vererbung von Properties? Nutzerrechte? Profile?

Was würde das lösen, was #186 nicht abdeckt?

> Vielleicht sogar ergänzend, alternativ mit einer kleinen Liste an abstrakteren Projekt-Typ-Templates, z.b. "Tracker-Assisted Cut", "Video-Download Centric". Vermutlich gerade am Anfang sinnvoll wenn noch keine oder nur wenige Studio-Projekte vorhanden sind.

Project Templates mit Properties hatten wir tatsächlich schon mal diskutiert (und auch befürwortet), siehe #186.

**Comment by CodeFreezr** [comment]
Ja! #186 klingt schon sehr nach der Projekt-Typ Template Idee. Brauchen wir dann hier an dieser Stelle nicht weiter vertiefen.

Eine Unterscheidung zwischen einem Event-Projekt und Studio-Projekt ermöglicht es unterschiedliche Vorgehensweisen wie http-tracker-flow, import-tool, fuse-tracker-flow, scp, etc. pp. vor allem bei verteilten Events unter einen Hut zu bringen.

Im Endeffekt könnte dieses Ticket #243 als ein UseCase für #186 verstanden werden.

**Comment by saerdnaer** [comment]
Ich dachte jetzt tatsächlich primär an eine Vererbung der Properties. Beim rC3-2020 hatten wir an die 10 Projekte, wenn sich da nen Property während der Veranstaltung bzw. nach dem Initialen kopieren ändert musste man in jedes einzelne Projekt und den Wert dort anpassen. Oder man greift halt direkt via SQL auf die Postgres zu... 🙈
